perllib/FixMyStreet/App/Controller/Report/New.pm

```diff
@@ -936,6 +936,13 @@ sub process_report : Private {
             $report->non_public( 1 );
         }
 
+        #Get a list of custom form fields we set and add them into @extra
+        if ($c->cobrand->form_extras){
+            foreach my $form_value ($c->cobrand->form_extras){
+                push @extra, { $form_value => $c->get_param_list($form_value)};
```

Comment:
I don't believe `get_param_list` is what you want here, as it returns a list value, which will potentially create extra fields in this hashref. e.g. if `$form_value` was `foo` and you had a URL with `foo=1&foo=2&foo=3` then this would end up with `{ foo => 1, 2 => 3 }` being pushed on to `@extra`. I imagine `get_param` should be good enough here, unless you had a specific reason otherwise?